ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -200,12 +223,15 @@ module Ldap = struct
       ; password_expired= logand user_account_control passw_expire_bit <> 0l
       }
 
-  let query_user sid domain_netbios kdc =
+  let env_of_lookup domain_netbios =
```

Comment:
The name could be misleading. This is the environment for the lookup, not of the lookup. I'd call it `krb5_config` because it defines the environment variable `KRB5_CONFIG`.